charts/newrelic-infrastructure/README.md

```diff
@@ -114,6 +114,7 @@ integrations that you have configured.
 | common.config.interval | duration | `15s` (See [Low data mode](README.md#low-data-mode)) | Intervals larger than 40s are not supported and will cause the NR UI to not behave properly. Any non-nil value will override the `lowDataMode` default. |
 | common.config.namespaceSelector | object | `{}` | Config for filtering ksm and kubelet metrics by namespace. |
 | containerSecurityContext | object | `{}` | Sets security context (at container level). Can be configured also with `global.containerSecurityContext` |
+| sink.http.probeTimeout | time.duration | 90s | The amount of time the scraper container to probe infra agent sidecar container before giving up and restarting during scraper agent starting. |
```

Comment:
Nit: from looking at the other rows, it looks like the styling for "90s" should be \`90s\` (see `common.config.interval` above).

Reply:
thanks. updated